src/ExComponent/ExComponent.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { Meta, Story } from '@storybook/react';
+import React from 'react';
+
+import { ExComponent } from './ExComponent';
+
+export default {
+  title: 'example component',
+  component: ExComponent,
+} as Meta<typeof ExComponent>;
+
+const Template: Story<typeof ExComponent> = (args) => <ExComponent {...args} />;
```

Comment:
may you check this error

Reply:
I have fix it but when commit it's add the () by default ! 
I will try to to fix it.